docs/docs/deployment/hosting/locally-api.md

```diff
@@ -447,6 +447,38 @@ Webpack compiles a front end build, sourcing `api/app/templates/index.html`. It
 The Django `collectstatic` command then copies all the additional static assets that Django needs, including
 `api/app/templates/webpack/index.html`, into `api/static`.
 
+## Rolling Back to a Previous Version of Flagsmith
```

Comment:
```suggestion
## Rolling back to a previous version
```

Reply:
I've done this to be consistent, but this is a title... it should be title case. But I'm not about to start updating all the titles!